src/index.js

```diff
@@ -12,38 +12,35 @@ const uint8ArrayToString = require('uint8arrays/to-string')
 const uint8ArrayConcat = require('uint8arrays/concat')
 
 const debug = require('debug')
-const log = debug('jsipns')
-log.error = debug('jsipns:error')
+const log = Object.assign(debug('jsipns'), {
+  error: debug('jsipns:error')
+})
 
-const ipnsEntryProto = require('./pb/ipns.proto')
+const {
+  IpnsEntry: ipnsEntryProto
+} = require('./pb/ipns.js')
 const { parseRFC3339 } = require('./utils')
 const ERRORS = require('./errors')
 
-const ID_MULTIHASH_CODE = multihash.names.id
+const ID_MULTIHASH_CODE = multihash.names.identity
```

Comment:
😱

Reply:
This used to be `.id` but apparently not flagged as a breaking change: https://github.com/multiformats/js-multihash/pull/66

Also, somehow we don't cover this in the tests 😱